### PR TITLE
[CPDRP-1165] Update automated ppt validation invitations and reminders

### PR DESCRIPTION
**Pull request by ngw**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDRP-1165

### CODE REVIEW

File: app/mailers/participant_mailer.rb [hunk 1]
```diff
@@ -3,21 +3,20 @@
 class ParticipantMailer < ApplicationMailer
   PARTICIPANT_REMOVED_BY_STI = "ab8fb8b1-9f44-4d27-8e80-01d5d70d22f6"
 
-  PARTICIPANT_ADDED_TEMPLATES = {
-    ect: "50ee41e5-06b9-41cf-9afd-d5bc4db356c4",
-    mentor: "e0198213-c09d-41aa-8197-b167e495e49d",
-    sit_mentor: "7e7d3fdb-41f5-4e04-a4ae-acf92e8fefe6",
+  PARTICIPANT_TEMPLATES = {
+    ect_cip: "i3bfad27e-e7af-4b53-94cb-af36362f43d8",
```
Comment (ethanmills) [inline app/mailers/participant_mailer.rb:7]: This `i` doesn't look right?

File: app/mailers/participant_mailer.rb [hunk 2]
```diff
@@ -3,21 +3,20 @@
...
+    sit_mentor_cip: "7aa80a9c-e486-42e8-92c8-7f970459d37d",
+    sit_mentor_fip: "0b7f850f-f26a-4e62-9fc5-17fbd8286e49",
   }.freeze
 
-  DETAIL_REMINDER_TEMPLATES = {
-    ect: "0bf633c3-54c9-4150-b1fb-57748376aed1",
-    mentor: "691eb49b-f23f-49dd-b799-f8efdd5e010f",
-    sit_mentor: "fcf85c00-6d96-48a6-bf27-ea09c61f0eee",
-  }.freeze
+  WHAT_EACH_PERSON_DOES_URL = "https://manage-training-for-early-career-teachers.education.gov.uk/pages/what-each-person-does"
```
Comment (ethanmills) [inline app/mailers/participant_mailer.rb:15]: ` Rails.application.routes.url_helpers.page_url(page: :"what-each-person-does", host: Rails.application.config.domain)` please, we don't hardcode the service URL. We may also want to add some UTM params to this?

File: app/mailers/participant_mailer.rb [hunk 3]
```diff
@@ -65,6 +68,6 @@ def add_details_reminder(participant_profile:)
   def template_type_for(profile)
     type = profile.participant_type
     type = :sit_mentor if type == :mentor && profile.user.induction_coordinator?
-    type
+    "#{type}_#{profile.school_cohort.cip? ? 'cip' : 'sit'}"
```
Comment (ethanmills) [inline app/mailers/participant_mailer.rb:71]:
```suggestion
    "#{type}_#{profile.school_cohort.cip? ? 'cip' : 'fip'}"
```
surely?

File: app/mailers/participant_mailer.rb [hunk 4]
```diff
@@ -26,6 +25,8 @@ def participant_added(participant_profile:)
         name: participant_profile.user.full_name,
         school_name: participant_profile.school.name,
         participant_start: new_user_session_url,
```
Comment (ethanmills) [inline app/mailers/participant_mailer.rb:27]: Do we want UTM params on this?

**Review by ethanmills — COMMENTED** [review]
Have you tested this?

File: app/mailers/participant_mailer.rb (hunk 1 above)
Comment (ngw) [reply app/mailers/participant_mailer.rb:7]: How did you catch that? :D

Comment (ethanmills) [reply app/mailers/participant_mailer.rb]: I was checking all the template IDs matched up with the names

**Review by ethanmills — APPROVED** [review]

**Pull request merged**